source/MaterialXGenShader/Util.h

```diff
@@ -123,6 +123,10 @@ vector<Vector2> getUdimCoordinates(const StringVec& udimIdentifiers);
 /// 0..1 space.
 void getUdimScaleAndOffset(const vector<Vector2>& udimCoordinates, Vector2& scaleUV, Vector2& offsetUV);
 
+/// Checks if output connected to shaderreeef input requires normals to be transformed from tangent 
```

Comment:
Shouldn't this method work for _any_ output of a NodeGraph?  If so, then we can omit the details here about the ShaderRef input (which will actually be replaced with a simpler shader node mechanism in MaterialX v1.38).

Reply:
Yes you are right! I will change the comment (and the typo)